app/controllers/reconcilliations_controller.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class ReconcilliationsController < ApplicationController
+
+  load_resource :order
+  load_resource :order_detail, through: :order
+
+  def destroy
+    if current_user.administrator?
```

Comment:
Usually, I would say we should strive for using the `Ability` for this kind of thing, but we don't really have a "real" model for this and this is just global managers, so I'm ok with it.